internal/core/model.rs

```diff
@@ -1093,6 +1096,10 @@ impl<C: RepeatedItemTree + 'static> Repeater<C> {
                 new_instances.push(new_instance);
             }
             if !new_instances.is_empty() {
+                for x in &mut indices_to_init {
+                    *x += new_instances.len();
+                }
+                indices_to_init.extend(0..new_instances.len());
```

Comment:
I did that because extend is faster. But I didn't think of the init order. But I don't think this is important since this would typically happen when scrolling and the existing item are already initialized